### ISSUE TITLE
Add possibility to select a fractional tollerance instead of the absolute tollerance 

### ISSUE THREAD

**Issue by bosonie** [greschd/aiida-optimize#49]
Some optimization engines, e.g. `NelderMead`, implement a check after each iteration to decide whether the optimization converged. The check looks at the difference between the best (two or more) results so far and compare it with a threshold st by the user.
This is perfectly fine if one knows roughly the magnitude of the quantity to optimize in advance, so it can select the threshold accordingly, however in high-throughput calculation this is not always the case. For this it would be nice to implement the possibility to set a FRACTIONAL tolerance instead of the absolute value of it. Of course this would be an optional possibility and retaining the current behavior as standard,

**Comment by greschd** [comment]
Yeah, I agree this would be nice in some cases. The details (which values to compare against etc.) probably depend on the specific optimization engine.
